docs/wiki/ModulesDesign.md

```diff
@@ -155,9 +88,44 @@ module server_databases 'databases/deploy.bicep' = [for (database, index) in dat
 
 Each module should come with a `.bicep` folder with a least the `nested_cuaId.bicep` file in it
 
+## Naming
+
+Use the following naming standard for module files and folders:
+
+- Module folders are in camelCase and their name reflects the main resource type of the Bicep module they are hosting (e.g. `storageAccounts`, `virtualMachines`).
+- Cross-referenced and providers resource modules are placed in the `.bicep` subfolder and named `nested_<crossReferencedResourceType>.bicep`
```

Comment:
```suggestion
- Cross-referenced and provider resource modules are placed in the `.bicep` subfolder and named `nested_<crossReferencedResourceType>.bicep`
```

Reply:
Extension resources... there is nothing in the official doc called provider resource modules.

Reply:
Replaced providers occurrences in this .md file. Probably we're using this naming in other files and they should be replaced as well.